js_modules/dagit/packages/core/src/runs/ComputeLogPanel.tsx

```diff
@@ -64,22 +59,24 @@ const ComputeLogsPanelWithKey: React.FC<ComputeLogPanelWithKeyProps> = React.mem
 
   const {isLoading, stdout, stderr} = useComputeLogs(runId, computeLogKey);
   const logData = ioType === 'stdout' ? stdout : stderr;
-  const downloadUrl = resolveDownloadUrl(rootServerURI, logData);
+  const downloadUrl = resolveDownloadUrl(rootServerURI, logData?.downloadUrl || null);
 
   return (
     <div style={{flex: 1, minHeight: 0, display: 'flex', flexDirection: 'column'}}>
       <ContentWrapper
-        logData={stdout}
+        content={stdout ? stdout.data : null}
+        path={stdout ? stdout.path : undefined}
+        downloadUrl={downloadUrl}
         isLoading={isLoading}
         isVisible={ioType === 'stdout'}
-        downloadUrl={downloadUrl}
         setComputeLogUrl={setComputeLogUrl}
       />
       <ContentWrapper
-        logData={stderr}
+        content={stderr ? stderr.data : null}
+        path={stderr ? stderr.path : undefined}
+        downloadUrl={downloadUrl}
```

Comment:
I think this works fine, but it's a bit confusing that the downloadUrl is only the right URL when `isVisible == true`. I wonder if it'd be a bit simpler to remove `logData`  and build two URLs, one for each ContentWrapper